commitizen/config/ini_config.py

```diff
@@ -0,0 +1,69 @@
+import configparser
+import json
+import warnings
+
+from .base_config import BaseConfig
+
+
+class IniConfig(BaseConfig):
+    warnings.simplefilter("always", DeprecationWarning)
```

Comment:
This is being executed always, even when a user is using `toml`, which is confusing. I'd move it to the `__init__`

Reply:
Yes, I added on another branch. I just cherry-pick it to this branch. Thanks!